tests/next_tests/unit_tests/program_processor_tests/runners_tests/dace_tests/transformation_tests/test_remove_point_view.py

```diff
@@ -129,3 +129,114 @@ def test_horizontal_map_fusion():
         == 0
     )
     assert util.compare_sdfg_res(ref=ref, res=res)
+
+
+def _make_sdfg_with_map_with_view_array(
+    N: str | int,
+) -> dace.SDFG:
+    shape = (N, N)
+    sdfg = dace.SDFG(util.unique_name("simple_map_with_view"))
+    state = sdfg.add_state(is_start_block=True)
+
+    for name in ["a", "out"]:
+        sdfg.add_array(
+            name=name,
+            shape=shape,
+            dtype=dace.float64,
+            transient=False,
+        )
+
+    a, out = (state.add_access(name) for name in ["a", "out"])
+    sdfg.add_symbol("horizontal_start", dace.int32)
+    sdfg.add_symbol("horizontal_end", dace.int32)
+
+    # First independent Tasklet
+    task1 = state.add_tasklet(
+        "task1",
+        inputs={
+            "__in0",  # <- `b[i, j]`
+        },
+        outputs={
+            "__out0",  # <- `tmp1`
+        },
+        code="__out0 = __in0 + 3.0",
+    )
+
+    # Now create the map using the above tasklets
+    mentry, mexit = state.add_map(
+        "simple_map",
+        ndrange={"i": f"0:{N}", "j": "horizontal_start:horizontal_end"},
+    )
+
+    tmp_access_node = state.add_access("tmp")
+    tmp_access_node.transient = True
+
+    sdfg.add_view(
+        "tmp",
+        shape=(1,),
+        dtype=dace.float64,
+    )
+
+    tmp_access_node2 = state.add_transient("tmp2", shape=shape, dtype=dace.float64)
+
+    state.add_edge(a, None, mentry, "IN_a", dace.Memlet("a[i, j]"))
+    state.add_edge(mentry, "OUT_a", task1, "__in0", dace.Memlet("a[i, j]"))
+    state.add_edge(task1, "__out0", tmp_access_node, None, dace.Memlet("tmp[0]"))
+    state.add_edge(tmp_access_node, None, tmp_access_node2, None, dace.Memlet("tmp[0]"))
+    state.add_edge(tmp_access_node2, None, mexit, "IN_out", dace.Memlet("tmp2[i, j]"))
+
+    mentry.add_scope_connectors("a")
+    mentry.add_in_connector("IN_a")
+    mexit.add_in_connector("IN_out")
+    mexit.add_out_connector("OUT_out")
+    state.add_edge(mexit, "OUT_out", out, None, dace.Memlet("out[i, j]"))
```

Comment:
```suggestion
    state.add_edge(mexit, "OUT_out", out, None, dace.Memlet("out[0:20, 0:20]"))
```
You are technically outside the map, thus the Map parameters do not exist anymore.
Why it works I do not know, but it is wrong.